// This is only an example to prove that tests actually run and succeed!
test("iOS AppDelegate.swift contains expected pnHandlerObj error call", async () => {
    // This is deleted in another branch pending merge, will go away when rebased
});